### PR TITLE
Remove compile.sh from doc

### PR DESCRIPTION
**Pull request by jinchengchenghh**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

**Comment by zhztheplayer** [comment]
Should `compile.sh` itself be removed, if not needed?

**Comment by jinchengchenghh** [comment]
As meeting discuss, not now. Someone may still use compile.sh. We will remove it after some time.

**Comment by zhztheplayer** [comment]
> As meeting discuss, not now. Someone may still use compile.sh. We will remove it after some time.

So could it be better to mark that file as deprecated (via adding comments)? So we don't forget removing it in the fullness of time?

**Review by zhztheplayer — APPROVED** [review]

File: dev/builddeps-veloxbe.sh [hunk 1]
```diff
@@ -84,9 +84,12 @@ fi
 
 ## compile gluten cpp
 cd $GLUTEN_DIR/cpp
-./compile.sh --build_velox_backend=ON --build_type=$BUILD_TYPE --build_velox_backend=ON \
-             --build_test=$BUILD_TESTS --build_benchmarks=$BUILD_BENCHMARKS --build_jemalloc=$BUILD_JEMALLOC \
-             --enable_hbm=$ENABLE_HBM --enable_s3=$ENABLE_S3 --enable_hdfs=$ENABLE_HDFS
+rm -rf build
+mkdir build
```
Comment (zhztheplayer) [inline dev/builddeps-veloxbe.sh:88]:
```suggestion
mkdir build
cd build
```

@jinchengchenghh can you make a quick fix on this? thanks.

**Pull request merged**